common/error.h

```diff
@@ -85,7 +85,7 @@ enum ErrorCode {
 class Error {
 protected:
 	ErrorCode _code; /*!< Error code. */
-	String _desc;    /*!< Error description. */
+	U32String _desc;    /*!< Error description. */
```

Comment:
See comment in `error.cpp`. I think we might want the description to be english here, and `String` is sufficient for that.